packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts

```diff
@@ -128,6 +132,28 @@ export async function generatePackage(
   } catch (e) {
     consola.error(e);
   }
+
+  // Check if anything generated clashes with names of our own types
+
+  const sanitizedOntology = sanitizeMetadata(ontology);
+  let clashCount = 0;
+  for (
+    const type of [
+      ...Object.keys(sanitizedOntology.actionTypes),
+      ...Object.keys(sanitizedOntology.interfaceTypes),
+      ...Object.keys(sanitizedOntology.objectTypes),
+      ...Object.keys(sanitizedOntology.queryTypes),
+      ...Object.keys(sanitizedOntology.sharedPropertyTypes),
+    ]
+  ) {
+    if (osdkTypes.has(type)) {
+      clashCount++;
+    }
+  }
+
+  if (clashCount > 0) {
+    consola.log("API names clashed: ", clashCount);
```

Comment:
Not actually sure if this will log it where we want

Reply:
If this cli uses consola then its probably okay? Maybe use `consola.error` or `consola.warn`? Do we think the user will actually see this when run from the UI?